### DISCUSSION TITLE
Do these Ebyte lora SPI modules tri-state?

### DISCUSSION THREAD

**Discussion by EliasA97** [jgromes/RadioLib#508] [General]
Hello, I'm making a custom pcb and I want to use the E22-400M30S spi module with a micro sd card. The thing is that I don't know if the ebyte lora module does tri-state on miso. I already added a tri-state chip for the sd card, but do I need to add for the lora?

**Comment by jgromes** [comment]
From what I've seen (opening up Ebyte modules), the SPI pins should be connected directly to the outside, with just a small series resistance, so there's nothing extra and whether the SPI pins are tri-stated or not depends entirely on the SX126x chip. I think that information should be in the datasheet - if not, try asking Semtech directly, I believe they had some developer forum.

**Reply by EliasA97** [reply]
Thank's

**Reply by EliasA97** [reply]
I think that's the answer to that.
![image](https://user-images.githubusercontent.com/56277356/162788984-d9985d74-0fb7-4152-a746-055bf23c8ac9.png)
